frontend/src/common/components/naviconsidebar/NavIconHomeButton.tsx

```diff
@@ -0,0 +1,27 @@
+import './NavIconHomeButton.scss'
+import { IconButton, Tooltip } from "@mui/material";
+import { Home } from "@mui/icons-material";
+import { useNavigate } from "react-router-dom";
+import * as routes from "../../../routes/constants";
+
+/**
+ * Displays the navigation icon for Home on the NavIconSideBar
+ */
+export const NavIconHomeButton = () => {
+    
+  const navigate = useNavigate()
+
+  return (
+      <div className="nav-icon-home-button-container">
+        <Tooltip arrow placement="left" title="Home">
+          <IconButton
+            size="medium"
+            color="secondary"
+            onClick={() => {navigate(routes.HOME)}}
```

Comment:
For the Home Button, it should navigate to routes.IDIR_WELCOME

Reply:
ok thanks!